storops/vnx/resource/system.py

```diff
@@ -116,6 +116,20 @@ def _init_file_cli(self):
                             self._file_username,

```

Comment:
Do you mean cleaning up the queue? I did not putting any gabage on the VNXSystem,